lib/query-interface.js

```diff
@@ -1244,6 +1254,7 @@ class QueryInterface {
    * @param {Array}   optionsArray  Extra-options for creation
    * @param {Object}  [options]     query options
    * @param {boolean} options.force If force is true, any existing functions with the same parameters will be replaced. For postgres, this means using `CREATE OR REPLACE FUNCTION` instead of `CREATE FUNCTION`. Default is false
+   * @param {Array}   options.variables List of declared variables
```

Comment:
I think this little documentation deserves improvement

```suggestion
   * @param {Array<Object>}   options.variables List of declared variables. Each variable should be an object with string fields `type` and `name`, and optionally having a `default` field as well.
```